SpringProject/src/main/java/com/gradproject2019/conferences/service/ConferenceServiceImpl.java

```diff
@@ -75,4 +85,12 @@ private void checkConferenceExists(Long conferenceId) {
             throw new ConferenceNotFoundException();
         }
     }
+
+    private void checkUserAuthorised(UUID token) {
+        try {
+            authServiceImpl.checkTokenExists(token);
+        } catch (TokenNotFoundException e) {
+            throw new InvalidCredentialsException();
```

Comment:
I think this one requires a new exception, something like UserUnauthorizedException. InvalidCredentialsException is already used for logging in with invalid credentials.